packages/configurationmap/components/ConfigurationMap.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+// import { style } from "../style";
+import { Table, Column } from "../../";
+
+export interface ConfigurationMapProps {
+  data: { [key: string]: string };
+}
+
+const cellRenderer = column => data => <span>{data[column]}</span>;
```

Comment:
This is a problem for rerenders: Every time data changes, we are going to have a new method being passed to the cell renderer, making every part below that rerendered. 

```suggestion
const cellRenderer = column => data => <span>{data[column]}</span>;
+ const keyRenderer = cellRenderer(0);
+ const valueRenderer = cellRenderer(1);
```

Making the references static would solve the problem